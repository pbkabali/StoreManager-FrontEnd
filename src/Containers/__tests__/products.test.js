import React from "react";
import Products from "../products";
import { shallow } from "enzyme";

describe("Product container tests", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<Products />);
    expect(wrapper).toMatchSnapshot();
  });
});
