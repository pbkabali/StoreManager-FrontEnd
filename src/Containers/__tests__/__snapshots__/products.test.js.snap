// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product container tests renders correctly 1`] = `ShallowWrapper {}`;
